cpu/riscv_common/periph/plic.c

```diff
@@ -30,8 +30,12 @@
 #include "plic.h"
 
 /* Local macros to calculate register offsets */
-#define _REG32(p, i) 			(*(volatile uint32_t *) ((p) + (i)))
-#define PLIC_REG(offset) 		_REG32(PLIC_CTRL_ADDR, offset)
+#ifndef _REG32
+#define _REG32(p, i)            (*(volatile uint32_t *)((p) + (i)))
+#endif
+#ifndef PLIC_REG
+#define PLIC_REG(offset)        _REG32(PLIC_CTRL_ADDR, offset)
+#endif
```

Comment:
I added these `#ifndef`s as both macros are also defined in vendor files and the reformatting led to redefinition errors as the whitespacing is not the same anymore. I hope this approach is feasible.